### ISSUE TITLE
How to restrict search to only return published documents?

### ISSUE THREAD

**Issue by Shelagh-Lewins** [matteodem/meteor-easy-search#381]
How can I make Easy Search secure by ensuring that the server only returns published documents in results?

**Comment by matteodem** [comment]
This can be accomplished by using `Minimongo` which does not create separate publications for search: http://matteodem.github.io/meteor-easy-search/docs/engines/.

**Comment by Shelagh-Lewins** [comment]
Thank you, that seems to work perfectly and is exactly what I want.

Maybe the first example shown in the readme should use Minimongo instead of MongoDB? I started by copying that example which uses MongoDB but doesn't include any warning or information about how to restrict search results and keep your data secure. I didn't realise for a while that I had opened a security hole, and it wasn't immediately obvious to me why (though I get it now, thank you again for all your help).

Am I right in thinking that if somebody wants to search on the server with MongoDB, they should use a selector to restrict what documents are returned? Again it might be worth putting a simple example of this in the readme or the getting started page.
